src/metastore.c

```diff
@@ -462,6 +464,8 @@ usage(const char *arg0, const char *message)
 "  -E, --remove-empty-dirs  Remove extra empty directories\n"
 "  -g, --git                Do not omit .git directories\n"
 "  -f, --file=FILE          Set metadata file (" METAFILE " by default)\n"
+"  -O, --owner              Owner to use when internal check fails.\n"
+"  -G, --group              Group to use when internal check fails.\n"
```

Comment:
I'm not sure I like this option names, they're too generic.
To avoid being misused, I would possibly not have short options for them, only long ones.
Descriptions are also too generic. Internal check is simply user/group ID existence in system's user/group database.

Suggestion:
- `--owner-for-unknown-uid` - Owner to store when user ID does not exist in system database
- `--group-for-unknown-gid` - Group to store when group ID does

Why we need to be so specific? Here we're dealing with storing.
But there is another case, when dealing with applying, that we may want to handle in future.
Like `--uid-for-unknown-owner` and `--gid-for-unknown-group` (or maybe: `--owner-for-unknown-owner` and `--group-for-unknown-group`), and they are not necessarily meant to have same arguments.

Using same -O and -G options for both storing and applying would be error-prone.